src/app/static/src/app/types.ts

```diff
@@ -336,3 +336,34 @@ export interface DownloadIndicatorPayload {
     filename: string
 }
 
+export interface Field {
+    key: string,
+    label?: string
+    sortable: boolean,
+    sortByFormatted: boolean
+}
+
+export interface GenericResponse<T> {
+    data: T | null
+    errors: {
+        error: string
+        detail: string | null
+        key?: string
+        trace?: string[]
+        [k: string]: any
+    }[]
+    version?: {
+        hintr: string
+        naomi: string
+        rrq: string
+        [k: string]: any
+    }
+}
+
+export interface StepperNavigationProps {
+    back: () => void
+    backDisabled: boolean
+    next: () => void
+    nextDisabled: boolean
+}
```

Comment:
Typescript upgrade complained because data type of Response from generated file is not Generic and this is required in `apiService` as seen in the code changes.

`StepNavigationProps` had a build error after Typescript upgrade, probably because of circular dependencies so moving it to types file seems like a reasonable thing to do.